src/dmd/target.d

```diff
@@ -25,6 +25,7 @@
 
 module dmd.target;
 
+import dmd.astenums;
```

Comment:
Is this really needed after fixing the selective imports later in this file?

Reply:
It failed to compile, so I put it in.

Reply:
Same question as @ibuclaw: What was the error ? Either this line is needed, and the changes on lines 1023 and 1373 are redundant (the lines should be remove), or it's not needed because it's already covered by the aforementioned lines.

Reply:
Removing the line did work. I added the line due to undefined STC before the compiler later complained about the selective import being wrong.